source/Nuke.Common/Tooling/ToolPathResolver.cs

```diff
@@ -130,5 +144,21 @@ public static string GetPathExecutable(string pathExecutable)
                 .FirstOrDefault(File.Exists)
                 .NotNull($"Could not find '{pathExecutable}' via '{locateExecutable}'.");
         }
+
+        private static void WriteMissingPackageFile(IEnumerable<string> packageIds, string version)
+        {
+            var contents =
+                new[] { NukeBuild.BuildProjectDirectory.GlobFiles("*.csproj").FirstOrDefault()?.ToString() }
+                .Concat(packageIds.Select(x => $"{x}@{version ?? "latest"}"))
```

Comment:
Let's use `undefined` instead of `latest` here.

Reply:
done